devices/ble_hci/common-hal/_bleio/att.c

```diff
@@ -1009,25 +995,6 @@ void process_read_group_req(uint16_t conn_handle, uint16_t mtu, uint8_t dlen, ui
     }
 }
 
-int att_read_group_req(uint16_t conn_handle, uint16_t start_handle, uint16_t end_handle, uint16_t uuid, uint8_t response_buffer[]) {
-
-    typedef struct __packed {
-        struct bt_att_hdr h;
-        struct bt_att_read_group_req r;
-    } req_t;
-
-    uint8_t req_bytes[sizeof(req_t) + sizeof(uuid)];
-    req_t *req = (req_t *)req_bytes;
-
-    req->h.code = BT_ATT_OP_READ_GROUP_REQ;
-    req->r.start_handle = start_handle;
-    req->r.end_handle = end_handle;
-    req->r.uuid[0] = uuid & 0xff;
-    req->r.uuid[1] = uuid >> 8;
-
-    return send_req_wait_for_rsp(conn_handle, sizeof(req_bytes), req_bytes, response_buffer);
-}
-
```

Comment:
Please don't delete this; the _bleio HCI implementation is incomplete and this may be needed later. Let's just comment it out for now.